mteb/tasks/STS/por/SickBrSTS.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class SickBrSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="SICK-BR",
+        dataset={
+            "path": "eduagarcia/sick-br",
+            "revision": "0cdfb1d51ef339011c067688a3b75b82f927c097",
+        },
+        description="SICK-BR is a Portuguese inference corpus, aligned to and translated from SICK",
+        reference="https://huggingface.co/datasets/eduagarcia/sick-br",
+        type="STS",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["por-Latn"],
+        main_score="cosine_spearman",
+        date=None,
```

Comment:
Make sure to fill these out, tests won't pass otherwise

Reply:
Thanks @x-tabdeveloping.

Interestingly, some existing datasets don't seem to have these either: https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/STS/pol/PolishSTS.py#L21-L30

Reply:
Yes, I know :D It's an issue we're working on. If you look at the points table, we also offer points for filling in metadata on older tasks.
We would just like new tasks to include all metadata if possible.

Reply:
Understood, thanks @x-tabdeveloping!